source/System.cpp

```diff
@@ -966,6 +972,15 @@ int System::MinimumFleetPeriod() const
 
 
 
+const vector<RaidFleet> &System::RaidFleets() const
+{
+	static const vector<RaidFleet> EMPTY;
+	// If the system defines its own raid fleets then those are used in lieu of the government's fleets.
+	return noRaids ? EMPTY : (raidFleets.empty() && government) ? government->RaidFleets() : raidFleets;
```

Comment:
Actually, one final thing. I'd prefer to use parentheses for nested ternary operators, as I've had their low precedence mess with me before.
```suggestion
	return noRaids ? EMPTY : ((raidFleets.empty() && government) ? government->RaidFleets() : raidFleets);
```